recipes/hklpy2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hklpy2" %}
+{% set version = "0.0.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hklpy2-{{ version }}.tar.gz
+  sha256: 159033cc0b81f9ac6f803ce50d39363cb704f64bb99d829630d13eaf42a30efc
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [python_impl == 'pypy']
+  skip: true  # [py<310 or win or osx]
+
+requirements:
+  host:
+    - python
+    - setuptools-scm >=8
+    - setuptools >=64.0
+    - pip
+  run:
+    - python
+    - packaging
```

Comment:
This one doesn't seem to be to in the package metadata.

Reply:
As the CI reported, `ophyd` needs `packaging` but that is not added with the ophyd package.  I'll contact that package and add `packaging` to this project's requirements until ophyd is revised.

Reply:
It's in the ophyd source but not in the conda-forge [recipe](https://github.com/conda-forge/ophyd-feedstock/blob/b8e8016dad28515c4bf138eee26c5fe45772e68d/recipe/meta.yaml#L18-L29).

Reply:
done